harness/propertyHelper.js

```diff
@@ -73,6 +73,18 @@ function verifyProperty(obj, name, desc, options) {
     }
   }
 
+  if (Object.prototype.hasOwnProperty.call(desc, 'get')) {
+    if (desc.get !== originalDesc.get) {
+      failures.push('descriptor get should be ' + desc.get);
+    }
+  }
+
+  if (Object.prototype.hasOwnProperty.call(desc, 'set')) {
+    if (desc.set !== originalDesc.set) {
+      failures.push('descriptor set should be ' + desc.set);
+    }
+  }
```

Comment:
same for set, better to defer this until we write tests using it.

Reply:
5ac819f